tsconfig.lib.json

```diff
@@ -0,0 +1,30 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "outDir": "out-tsc/lib",
+    "target": "es2015",
+    "module": "es2015",
+    "moduleResolution": "node",
+    "declaration": true,
+    "sourceMap": true,
+    "inlineSources": true,
+    "importHelpers": true,
+    "types": [],
+    "lib": [
+      "dom",
+      "es2017"
+    ]
+  },
+  "angularCompilerOptions": {
```

Comment:
Why not immediately take out a separate file `tsconfig.json` and not inherit from it?

Reply:
angularCompilerOptions move to tsconfig.json